ansible-ipi-install/playbook.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: IPI on Baremetal Installation Playbook
+  hosts: provisioner
+  vars:
+    activation_key: "mykey"
+    org_id: "111111"
+    myuser: kni
+    version: "4.3.0-0.nightly-2019-12-09-035405"
+    pullsecret: 'ENTER SECRET HERE'
+  roles:
+    - role: node-prep
+      vars:
```

Comment:
we'd want the specific variables to be defined outside of the playbook

Reply:
All variables are defined and some default is set in the group_vars/all. However, for common variables that are likely to be modified or require to be modified it seemed the playbook.yml was the logical choice. I made my decision based on Ansible's doc: https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable